src/DotNetUnknown/Program.cs

```diff
@@ -12,12 +13,20 @@
         "[{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz}] [{Level:u3}] [{SourceContext}] [ThreadId {ThreadId}] {Message:lj}{NewLine}{Exception}")
 );
 
+builder.Services.AddControllers();
+builder.Services.AddExceptionHandler<GlobalExceptionHandler>();
+builder.Services.AddProblemDetails();
+
 builder.Services.AddTransient<LoggingUtils>();
 
+builder.Services.AddSingleton<GlobalExceptionHandler>();
 
 var app = builder.Build();
 
-// Optional: Adds middleware to log HTTP requests automatically via Serilog
 app.UseSerilogRequestLogging();
 
+app.MapControllers();
+app.UseRouting();
+app.UseExceptionHandler();
```

Comment:
The middleware sequence really matters, it may make a different result
```suggestion
app.UseExceptionHandler();
app.MapControllers();
app.UseRouting();
```